src/main/webapp/app/modules/provider/claim-records-modal.tsx

```diff
@@ -191,6 +201,38 @@ export class ClaimRecordsModal extends React.Component<IClaimRecordsModalProps,
     </div>
   );
 
+  turnOffExitModal = () => this.setState({ isExiting: false });
+
+  exitModal = numberOfClaimedRecords => (
```

Comment:
Rename this to something more list modalOnExit or sth like that, because exitModal is misleading.